client/rest-high-level/src/main/java/org/elasticsearch/client/watcher/DeactivateWatchRequest.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.elasticsearch.client.watcher;
+
+import org.elasticsearch.client.Validatable;
+import org.elasticsearch.client.ValidationException;
+import org.elasticsearch.protocol.xpack.watcher.PutWatchRequest;
+
+import java.util.Optional;
+
+public class DeactivateWatchRequest implements Validatable {
+    private final String watchId;
+
+    public DeactivateWatchRequest(String watchId) {
+        this.watchId = watchId;
+    }
+
+    public String getWatchId() {
+        return watchId;
+    }
+
+    @Override
+    public Optional<ValidationException> validate() {
+        ValidationException exception = new ValidationException();
```

Comment:
Lets move this validation into the constructor and remove this validate method. Still leave it Validatable, its just a relic :)